### ISSUE TITLE
Delete vehicle Feature

### ISSUE THREAD

**Issue by SM-Marc** [SM-BJS/StowMarines-Training-Map#25]
In the area of Alpha/Phoenix Garage the delete vehicle feature pops next to the gate (Flag) and i'm not sure if it is there on purpose in addition to its original place next to the orange barriers between the two buildings which is kinda tricky to find there. But the one at the gate seems off. I haven't tested the delete vehicle that pops next to the gate with a vehicle between the orange barriers but it doesn't delete the vehicle that is positioned at the gate where the feature shows.

PS: Sorry for the Late reply

**Comment by SM-BJS** [comment]
Okay so I've just looked into this, the only time I can get the interaction to appear near the flag and gate is when I look directly at the delete vehicle place (the correct one) and that's a result of Arma 3's super stretched addaction distance. I have also double checked to see if it's still working and it's giving me no problems. I'll look into limiting the addaction distance. Can you confirm that the issue was being able to interact at the flag?

**Comment by SM-Marc** [comment]
Affirm, the feature works totally fine at the correct place. The problem is being able to interact at the flag and as expected nothing happens there when you try and delete a vehicle.

**Comment by SM-BJS** [comment]
Okay, I've fixed the radius to 10m instead of 50m which limits it to only be intractable near the orange barriers. The fix will be released in version update 7.3. Thanks for reporting it.

The changes in code:

**Original**
```
this addAction["Delete Vehicle",{_spCheck = nearestObjects[repairBox,["landVehicle","Air","Ship"],10] select 0; 
if(!isNil "_spCheck") then {deleteVehicle _spCheck;};}];
```

**New**
```
this addAction
[
	"Delete Vehicle",
	{
		_spCheck = nearestObjects[repairBox,["landVehicle","Air","Ship"],10] select 0; 
		if(!isNil "_spCheck") then {deleteVehicle _spCheck;};
	},
	nil,
	1.5,
	true,
	true,
	"",
	"true",
	10,
	false,
	"",
	""
];
```
